articles/how-to-replicate-postgresql-database/index.md

```diff
@@ -0,0 +1,226 @@
+Replication is the act of reproducing or copying with something Several Relational Database Management Systems have different ways to achieve this. PostgreSQL being an open-source RDMS is what we are going to use in this article and show how replication is achieved.
+
+### Introduction
+
+For replication to be achieved in PostgreSQL there must be two servers that can be able to communicate with each other. This server will be identified as Master which is the Master server or the production server and the other one is the Slave server or Replica server or standby Server which will have a copy of master server data. The data is synchronized in such a way that if the Master server fails one can continue to run with the slave database without any problem.
+Replication is important to avoid Failover whereby the primary database fails and due to business continuity, there is a need for recovery. This recovery will depend on the replicated database created on Slave Server.
+
+### Why Replication
+
+1. Online Transaction Processing(OLTP) Performance- removing reporting query load from OLTP system improves both query time and transaction processing time
+2. Data migration- which comes about either through system deployment or change of database server hardware
+3. System Testing in Parallel- when upgrading a new system there need to make sure the new system works well with existing data hence need to test with production database copy before deployment
+4. Fault Tolerance- in case the main server fails the standby server can act as a server since the contained data  is the same
+
+For this to happen there must be communication between the two hosts or two servers that is through a network or internet as shown below.
+
+![LOCAL-EXAMPLE](/How-to-Replicate-PostgreSQL-database/local-network.png)
+
+![INTERNET-EXAMPLE](/How-to-Replicate-PostgreSQL-database/internet-network.png)
+
+### Installing PostgreSQL
+
+For this article, we will use Ubuntu 18.04 LTS and PostgreSQL 10. Several Linux distributions can be used as server operating systems. It is important to note that one can set up Replication in any operating system.
+
+Make sure you have installed the Linux Ubuntu server. To install PostgreSQL 10 in Ubuntu 18.04LTS one must follow the following steps. This must be done to both servers that is the Master Server and Slave Server.
+
+1. import PostgreSQL signing key by typing the following command in Terminal
+
+<br>
+```
+wget -q https://www.postgresql.org/media/keys/ACCC4CF8.asc -O- | sudo apt-key add -
+```
+
+2. Add PostgreSQL repository by typing the following command in terminal
+
+<br>
```

Comment:
Instead of using <br> as line breaks, it is better to use an extra line (it automatically inserts a line break). I've added a commit that removes the <br> tags